### PR TITLE
ms_paint

### PR DESCRIPTION
**Pull request by shamilaa**
DDG Community Kochi
#2127 

------
https://duck.co/ia/view/ms_paint_cheat_sheat

### CODE REVIEW

**Comment by daxtheduck** [comment]
@shamilaa Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/ms_paint.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
+{
+  "id": "mspaint_cheat_sheet",
+  "name": "MS Paint",
+  "description": "Keyboard shortcuts for MS paint",
+  "metadata": {
+    "sourceName": "Microsoft",
+    "sourceUrl": "https://en.wikipedia.org/wiki/Paint_(software)"
+  },
+  "aliases": [
+    "ms paint",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/ms_paint.json]: can you add `mspaint` as an alias?

File: share/goodie/cheat_sheets/json/ms_paint.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
+{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/ms_paint.json]: @shamilaa thanks for contributing! Please change the filename to `ms-paint.json`

It needs to use dashes (`-`) :+1:

File: share/goodie/cheat_sheets/json/ms_paint.json [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  "id": "mspaint_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/ms_paint.json]: @shamilaa looks like you missed an `_` -- this needs to match the `ID` on your IA page, `ms_paint_cheat_sheet`.

Your filename must then be `ms-paint.json`

File: share/goodie/cheat_sheets/json/ms_paint.json (hunk 2 above)
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/ms_paint.json]: Yup. Please do this.

**Comment by moollaza** [comment]
@shamilaa thanks again for all the hard work you've put into creating this MS Paint Cheat Sheet. You've done a great job so far! We at DuckDuckGo, and the DuckDuckHack community really appreciate you taking the time to contribute.

It looks like this still needs some work?  If you're stuck or need any help, I'd love to chat if you're around this week or next. Please message me on slack (@moollaza) or send me an email (moollaza@duckduckgo.com) if you'd be interested in talking. 

Thanks again!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=ms paint cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/ms_paint_cheat_sheat_mobile.png)

**Comment by moollaza** [comment]
@shamilaa I'm going to close this for now. If you'd like to finish it up later, I'd be happy to re-open the PR for you.

**Pull request closed without merging**